test/apps/angular-v12/src/app/app.component.ts

```diff
@@ -10,6 +10,7 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+/*eslint import/no-unresolved: [2, { ignore: ['@okta/okta-angular$'] }]*/
```

Comment:
nit: maybe just another .eslintrc under test/apps, which should be able to inherit rules from the root